### DISCUSSION TITLE
Attempt to decode JSON with unexpected mimetype

### DISCUSSION THREAD

**Discussion by Domi31tls** [open-webui/open-webui#7895] [Issues]
# Bug Report

## Installation Method

Docker v0.4.8

- API Mistral : https://api.mistral.ai/v1
- Model mistral_large
- Tika 

Config rag :

![image](https://github.com/user-attachments/assets/c9b103ae-c0b8-435a-a20b-f69cc256a4fb)

 

- **Operating System:**ubuntu 24.04
- **Browser :chrome

## Description

When I download a document, I get an error in the logs. Additionally, the responses are very slow to arrive.
I've the same problémes with  other models with ollama (localhost)

## Reproduction Details

![image](https://github.com/user-attachments/assets/a72e6a1e-f35a-4d68-bcc4-6f054c876c3a)

## Logs and Screenshots :+1: 
**INFO  [open_webui.apps.openai.main] get_all_models()
ERROR [open_webui.apps.openai.main] Connection error: 404, message='Attempt to decode JSON with unexpected mimetype: ', url='http://10.23.0.3:9998/models'**
INFO  [open_webui.apps.ollama.main] get_all_models()
INFO  [open_webui.apps.retrieval.utils] query_doc_with_hybrid_search:result [[{'Content-Type': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'created_by': '64d1d537-db47-4edd-a95c-5725dcc4e7fb', 'embedding_config': '{"engine": "openai", "model": "mistral-embed"}', 'file_id': '05fb5fc6-84de-4efb-a590-c5abf6e5ac7e', 'hash': 'efcb5a4e...................................................................
 ........................................
**INFO  [open_webui.apps.openai.main] get_all_models()
ERROR [open_webui.apps.openai.main] Connection error: 404, message='Attempt to decode JSON with unexpected mimetype: ', url='http://10.23.0.3:9998/models'**
INFO  [open_webui.apps.ollama.main] get_all_models()
INFO:     10.10.112.2:36282 - "POST /api/chat/completions HTTP/1.1" 200 OK

Thank for your help

**Comment by gaby** [comment]
@Domi31tls Ever found a solution to this ?

**Comment by SKJoy** [comment]
And this bug still exists :( It usually occurs when the conversation goes very long, and usually resolves if you delete a few previous messages! Looks like a capacity issue?

**Comment by skrenes** [comment]
I think this is a parsing issue. While I have a very long chat, it started adding double quotations to proposed solutions and then this error started showing up.
